tests/core/test_check_manifest.py

```diff
@@ -35,9 +35,12 @@ def test_check_allowed_filepaths_pass(scroot, monkeypatch):
     chip.set('input', 'verilog', os.path.join(scroot, 'examples', 'gcd', 'gcd.v'))
     chip.load_target("freepdk45_demo")
 
-    # run an import just to collect files
-    chip.set('option', 'steplist', 'import')
-    chip.run()
+    # collect input files
+    workdir = chip._getworkdir(step='import', index='0')
+    os.makedirs(workdir)
+    os.chdir(workdir)
+    chip._collect('import', '0')
+    os.chdir(chip.cwd)
```

Comment:
Should we store the previous dir instead of using `chip.cwd` to be safe, since `os.chdir` is a global interpreter setting? Same for the following test.

Reply:
Yeah that seems safer and more explicit, fixed. I'm curious now if `pytest` has some mechanism to isolate a changed working directory between tests - I wanted to see if they have an official way to do this, but looks like their monkeypatch library just uses os.chdir as well: https://docs.pytest.org/en/7.1.x/_modules/_pytest/monkeypatch.html#MonkeyPatch.chdir. 

Either way, I think this is a good change!